services/app-api/src/authn/cognitoAuthn.ts

```diff
@@ -138,6 +138,7 @@ export function userTypeFromAttributes(
             givenName: attributes.given_name,
             familyName: attributes.family_name,
             stateAssignments: [],
+            divisionAssignment: attributes.division,
         })
```

Comment:
Will `localAuthn.ts` need similar logic?

Reply:
Doesn’t look like it, we aren’t parsing the user out the same way.